src/main/webapp/js/lockable-resources.js

```diff
@@ -7,6 +7,26 @@ function find_resource_name(element) {
   return resourceName;
 }
 
+function changeQueueOrder(button, queueId) {
+
+  dialog
+      .prompt(i18n("queue-title"), {
+        message: i18n("queue-message", queueId),
+        minWidth: "450px",
+        maxWidth: "600px" })
+      .then(
+        (newPosition) => {
+          var form = document.createElement("form");
+          form.setAttribute("method", "POST");
+          form.setAttribute("action", "changeQueueOrder?id=" + queueId + "&index=" + newPosition);
+          crumb.appendToForm(form);
+          document.body.appendChild(form);
+          form.submit();
+          notificationBar.show(i18n("queue-on-success"), notificationBar.SUCCESS);
```

Comment:
After successfully changing the queue order, the http redirect dumps us back to the "Resources" tab instead of the "Queue" tab.
This is annoying because you cannot immediately see the effects of the order change.

The same goes for refreshing the page: how can I update the "Queue" table without leaving the page? Hitting F5/refresh will navigate me back to the "Resources" tab.

Reply:
I think for that is other one issue, but I can try to fix it also in this PR

Reply:
It shall be fixed in bootstrap plugin. But to do not wait for the fix (may be endless), I provide the fix on this plugin side. And I like the feature